bitnami/tomcat/values.yaml

```diff
@@ -269,3 +269,11 @@ readinessProbe:
   periodSeconds: 51
   failureThreshold: 3
   timeoutSeconds: 3
+
+## Additional labels to apply to all resources
+labels: {}
+#   foo: bar
+
+## Additional labels to use on deploy.spec.selector.matchLabels and svc.spec.selector
+matchLabels: {}
```

Comment:
Why would it be useful to modify the current matchLabels with extra ones?

Reply:
I am not sure. At the time I thought I saw other charts did this so I figured might as well while Im here, but looking now I can't find any so I'll remove this